token-client/src/main/java/com/sap/cloud/security/mtls/SSLContextFactory.java

```diff
@@ -91,8 +91,17 @@ public SSLContext create(String x509Certificates, String privateKey)
 	 */
 	public SSLContext create(ClientIdentity clientIdentity) throws GeneralSecurityException, IOException {
 		assertNotNull(clientIdentity, "clientIdentity must not be null");
-		assertHasText(clientIdentity.getCertificate(), "clientIdentity.getCertificate() must not return null");
-		assertHasText(clientIdentity.getKey(), "clientIdentity.getKey() must not return null");
+		try {
```

Comment:
I think this is try/catch is redundant because the same checks are performed inside createKeyStore which is the first statement following these lines